src/test/java/io/vertx/util/Container.java

```diff
@@ -0,0 +1,18 @@
+package io.vertx.util;
+
+public class Container<T> {
```

Comment:
don't introduce this artificial class, instead use `AtomicReference` in your test which acts as a decent container, furthermore `AtomicReference` provides the same visibility semantics than volatile.

Reply:
Yes.